app/main/views/admin_manager.py

```diff
@@ -28,3 +31,33 @@ def manage_admin_users():
 
     return render_template("view_admin_users.html",
                            admin_users=admin_users)
+
+
+@main.route('/admin-users/invite', methods=['GET', 'POST'])
+@role_required('admin-manager')
+def invite_admin_user():
+    form = InviteAdminForm(request.form)
+
+    if request.method == 'POST' and form.validate_on_submit():
+        email_address = form.data.get('email_address')
+        role = form.data.get('role')
+
+        send_user_account_email(
+            role,
+            email_address,
+            '08ab7791-6038-4ad2-9560-740bbcb675b7',
```

Comment:
Can we pull this out as something like `NOTIFY_TEMPLATE_ID = "..."` so it's clear what this mysterious string is? 😸

Reply:
Done 👍